specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/examples/ManagedInstanceCreateMax.json

```diff
@@ -0,0 +1,99 @@
+{
+  "parameters" : {
+    "subscriptionId" : "20D7082A-0FC7-4468-82BD-542694D5042B",
+    "resourceGroupName" : "testrg",
+    "managedInstanceName" : "testinstance",
+    "api-version" : "2018-06-01-preview",
+    "parameters" : {
+      "tags" : {
+        "tagKey1" : "TagValue1"
+      },
+      "location" : "Japan East",
+      "sku" : {
+        "name" : "GP_Gen5",
+        "tier" : "GeneralPurpose"
+      },
+      "properties": {
+        "administratorLogin": "dummylogin",
+        "administratorLoginPassword": "Un53cuRE!",
+        "subnetId": "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Network/virtualNetworks/vnet1/subnets/subnet1",
+        "vCores": 8,
+        "storageSizeInGB": 1024,
+        "licenseType": "LicenseIncluded",
+        "collation": "SQL_Latin1_General_CP1_CI_AS",
+        "dnsZonePartner": "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Sql/managedInstances/testinstance",
+        "publicDataEndpointEnabled": false,
+        "proxyOverride": "Redirect",
+        "timezoneId": "UTC",
+        "instancePoolId": "/subscriptions/20D7082A-0FC7-4468-82BD-542694D5042B/resourceGroups/testrg/providers/Microsoft.Sql/instancePools/pool1"
```

Comment:
is it possible to include example showing usage of `MaintenanceWindowSettings` property?

Reply:
Not needed as we're removing this property.